src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -19,11 +20,12 @@ public Movie add(Movie movie) {
 
     @Override
     public Movie get(Long id) {
-        return movieDao.get(id).get();
+        return movieDao.get(id).orElseThrow(() ->
+                new NoSuchElementException("Cinema Hall not found."));
```

Comment:
The exception message in the `get` method incorrectly refers to 'Cinema Hall' instead of 'Movie'. Consider updating the message to 'Movie not found.' to accurately reflect the entity being handled.